tools/perl-version-convert

```diff
@@ -70,8 +70,11 @@ function convert_version
 	# be converted to 0.XX.YY, but it was already integrated as 0.XXYY, so
 	# we need to follow it for now.
 	#
+	# convert "Mail-AuthenticationResults" from v.YYYYMMDD to v.YYYY.M.DD
```

Comment:
Strictly speaking, this comment is not needed, because the conversion is obvious.  All comments above explains some unexpected or strange conversions.  Anyway, if you prefer to keep it here, then please change `.M.` to `.MM.`.